Examples/MagicWeather/android/app/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "m4trixdevlimited.com.flutter_magic_weather_app"
+        applicationId "com.revenuecat.sampleapp"
```

Comment:
I missed a couple of these when fixing the package names in #170 🤦 
I tested that it works correctly for both android and ios